libres/lib/analysis/enkf_linalg.cpp

```diff
@@ -35,10 +35,28 @@ void enkf_linalg_genX3(matrix_type *X3, const matrix_type *W,
     matrix_free(X2);
 }
 
+/*
```

Comment:
C++ comments tend to have the following form:
```c++
/**
 * [ TEXT ]
 */
```

We are sort of trying to use doxygen for documentation generation, although it's not actively being generated. We can still attempt to write docs in that format though.

Reply:
Fixed, thanks.